src/common/clamp.ts

```diff
@@ -0,0 +1,11 @@
+export const clamp = (number: number, boundOne: number, boundTwo: number) => {
```

Comment:
I think this approach could fail if `boundTwo` is equal to `0`.

What about following the minimum and maximum approach? probably this way we can ensure if a number is correctly contained between 2 bounds, something like:

```javascript
export const clamp = (number: number, boundOne: number, boundTwo: number): number => {
    // Determine the minimum and maximum bounds
    const min = Math.min(boundOne, boundTwo);
    const max = Math.max(boundOne, boundTwo);

    // Return the clamped value
    return Math.max(min, Math.min(number, max));
};
```

`Math.min(number, max)`: Ensures the number does not exceed the maximum bound.
`Math.max(min, ...)`: Ensures the number does not fall below the minimum bound.